undertow/src/main/resources/schema/wildfly-undertow_13_0.xsd

```diff
@@ -74,8 +74,8 @@
         <xs:attribute name="send-buffer" type="xs:int"/>
         <xs:attribute name="tcp-backlog" type="xs:int" default="10000"/>
         <xs:attribute name="tcp-keep-alive" type="xs:boolean"/>
-        <xs:attribute name="read-timeout" type="xs:long"/>
-        <xs:attribute name="write-timeout" type="xs:long"/>
+        <xs:attribute name="read-timeout" type="xs:long" default="60000"/>
+        <xs:attribute name="write-timeout" type="xs:long" default="60000"/>
```

Comment:
Version 13 was introduced in WF27.  Modifications to XSDs are generally done in a new version.
I've patched this here: https://github.com/wildfly/wildfly/pull/16455